src/tmpServer.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+import { features as countries } from './fullCountries.json';
```

Comment:
this isn't necessarily related to this specific line but... I think we agreed to remove some countries with insufficient data (UMI - no latlon data / HMD - no capital, continent, population 0), should that change go in with this fix?

Reply:
We will remove the in the blitz.